sinks/gcm/driver.go

```diff
@@ -42,6 +42,15 @@ type gcmSink struct {
 	// TODO(vmarmol): Also store labels?
 	// Map of metrics we currently export.
 	exportedMetrics map[string]sink_api.MetricDescriptor
+
+	// TODO(vmarmol): Garbage collect these.
+	// Time we last exported a particular timeseries.
+	lastExportedTime map[lastExportedKey]time.Time
```

Comment:
We already do, but this is used here (again) because GCM rounds to seconds. We can enforce that on all backends or re-do this here where it is enforced.